packages/ra-ui-materialui/src/input/AutocompleteInput.js

```diff
@@ -444,6 +444,7 @@ export class AutocompleteInput extends React.Component {
             source,
             className,
             options,
+            autosuggestProps = {}
```

Comment:
We should use the same convention as for the other input/field components: pass all unknown props to the root component, here `Autosuggest`. 

I suggest replacing this prop by `...rest`

Reply:
done